spec/workflow/context_spec.rb

```diff
@@ -214,5 +214,38 @@
     it "doesn't expose credentials" do
       expect(ctx_creds.to_h).not_to include("Credentials")
     end
+
+    # NOTE: introduced for specs
```

Comment:
I don't think we need to say why we introduced it, being able to compare `Context` can be helpful outside of just specs